contract/src/DaiRadicleCloud.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: GPL-3.0-only
```

Comment:
Why does DAI have its own contract? Shouldn't we only need ERC20 support?

Reply:
ERC-20 has this broken `approve(address,amount)` pattern, which is both a security risk and extra gas cost. Other tokens such as Dai (with `permit`) and USDC (with `transferWithAuthorization`) have implemented their own workarounds, thus we use separate contracts for each.

Reply:
Gotcha, this is definitely worth adding as a comment at the top of one of the files.